nestor_api/lib/docker.py

```diff
@@ -0,0 +1,82 @@
+"""Docker library"""
+
+import nestor_api.lib.config as config
+import nestor_api.lib.git as git
+import nestor_api.lib.io as io
+from nestor_api.utils.logger import Logger
+
+
+def build(app_name, context):
+    """Build the docker image of the last version of the app
+        Example:
+        >>> app_name = {"my-app"}
```

Comment:
I thought `app_name` would be a string. Is a set what you a expect as an argument ?